tests/ci_folder_parsing.py

```diff
@@ -29,11 +34,16 @@
                        type=str,
                        help="The Token associated with the class, such as 'asr1k'",
                        default=None)
+my_parser.add_argument('-n', "--number",
+                       type=str,
+                       help="The specific unittest we want to run, such as '25'",
+                       default=None)
 args = my_parser.parse_args()
 
 _os = args.operating_system
 _class = args.class_name
 _token = args.token
+_number = args.number
```

Comment:
Does it make sense to ensure if _number is assigned, that so us _os and _class?

Reply:
* for _n, I've changed the type to be an integer to ensure something like ``10.5`` is not passed through
* for _os, I've added a check to ensure the argument is provided
* for _class, I've added a check to provide an error if -n is provided without -c